crates/cargo-bsp/src/cargo_communication/cargo_types/cargo_command.rs

```diff
@@ -17,13 +17,22 @@
 //! `--show-output -Z unstable-options --format=json` for `cargo test`
 //! (only with `+nightly`). These flags format information about the tests to JSON and
 //! allows additional information, such as when each single tests started and finished,
-//! their stdout and stderr.
+//! their stdout and stderr
+//!
+//! `--workspace --all-targets -Z unstable-options --keep-going` for `cargo check`.
+//! `--all-targets` is needed to compile:
+//! - build scripts even if a crate doesn't contain library or binary targets,
+//! - dev dependencies during build script evaluation
+//! `--keep-going` is needed to compile as many proc macro artifacts as possible.
 
 use serde_enum_str::{Deserialize_enum_str, Serialize_enum_str};
 use std::path::Path;
 
-use crate::project_model::target_details::CargoTargetKind::Lib;
+use crate::project_model::target_details::CargoTargetKind::{
+    Cdylib, Dylib, Lib, ProcMacro, Rlib, Staticlib,
```

Comment:
nie lepiej CargoTargetKind::*?

Reply:
Nie są to wszystkie kind, więc w sumie nie wiem co lepsze

Reply:
może tak zostać, ma to małe znaczenie. Jedynie względy estetyczne